.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -0,0 +1,83 @@
+name: 🐛 Bug report
+description: Create a report to help us improve
+title: "[BUG] <title>"
+labels: ['bug']
+body:
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Related component
+      description: Choose one specific OpenSearch component your problem belongs to.
+      multiple: false
+      options:
+        - Search:Resiliency
+        - Search:Performance
+        - Search:Query Capabilities
+        - Search:Query Insights
+        - Search:Aggregations
+        - Search:Remote Search
+        - Search:Relevance
+        - Search:Searchable Snapshots
+        - Search
+        - Indexing:Replication
+        - Indexing:Performance
+        - Indexing
+        - Storage:Snapshots
+        - Storage:Performance
+        - Storage:Durability
+        - Storage:Remote
+        - Storage
+        - Cluster Manager
+        - Extensions
+        - Build Libraries & Interfaces
+        - Plugins
```

Comment:
Do these show up in the dropdown in the order given here? If so, can we sort these alphabetically? 

I agree with leaving the "general" labels below the more specific ones (i.e. `Search` would come after `Search:Searchable Snapshots`), but otherwise, I think it would be a lot easier if the whole thing were sorted. Thanks!